client/src/actions/index.js

```diff
@@ -120,40 +137,53 @@ export const login = (email, password, history) => {
 
     axios
       .post(`${ROOT}/users/login`, { email, password })
-      .then(({ data }) => {
-        dispatch({ type: AUTH_ERROR_RESET });
-        localStorage.setItem('token', data.token);
+      .then(response => {
+        // dispatch({ type: AUTH_ERROR_RESET });
+        // - Update state to indicate user is authenticated
         dispatch({ type: AUTH_LOGIN_SUCCESS, payload: email });
-        dispatch({ type: AUTH_LOGIN_FINISH });
-        history.push('/');
+        localStorage.setItem('token', response.data.token);
```

Comment:
tokens are set in cookies, so there's no need to set this. if you go to `dev tools > cookies > localhost:3000 > connect.sid`, this is the cookie for user auth

Reply:
removed this line and other token related lines